src/interfaces.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * A module for common interfaces
+ * @module interface
+ */
+
+/**
+ * Standard error function
+ *
+ * @memberof module:interface
+ * @param {Object} input The input object
+ * @param {Object} input.type The error type
+ * @param {Object} input.message The error message
+ * @param {Object} input.errors The errors array
```

Comment:
This should be `@param {Object[]}` which expects an array with objects of this type:

```js
{
   param, 
  msg,
  location
}
```

define a custom type and use this type in the param definition:
https://jsdoc.app/tags-typedef.html